jax/experimental/jax2tf/examples_eval/all_examples.py

```diff
@@ -117,8 +117,12 @@ def _flax_examples():
       'ppo':
           ModuleSpec(
               module_path='ppo.models.ActorCritic',
-              input_shape=(1, 8, 8, 4),
-              module_kwargs=dict(num_outputs=2)),
+              # TODO(marcvanzee): We get numerical differences if we run this
```

Comment:
I am wondering why not let it fail? I.e., why adjust the inputs so that it passes?

I am also wondering if we should try to run by hand multiple valid combinations of parameters, maybe there are more failures?

Reply:
I changed it to these input shape since these were the ones used by the example. But I agree we can just let them fail, which will make it easier to fix them. I have also reverted the other change in shapes (in SST2) so that one is failing now as well.

As a follow-up, I think it would be useful to change `input_shape` to a list, where we could add more shapes that we want to test.

Reply:
I agree, I changed it so it is now failing again.

Multiple combinations of inputs is a good idea, I can do that as a follow-up. In the meantime I've now extended it so that it runs 5 times with different input data, which did lead to another example failing, which is a good sign.